mon-pix/app/mixins/secured-route-mixin.js

```diff
@@ -0,0 +1,22 @@
+import { inject as service } from '@ember/service';
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+
+  router: service(),
+  session: service('session'),
+  currentUser: service(),
+  authenticationRoute: 'login',
+
+  beforeModel(transition) {
+    if (!this.session.get('isAuthenticated')) {
+      this.session.set('attemptedTransition', transition);
+      this.router.transitionTo(this.authenticationRoute);
```

Comment:
Je pense qu'on peut encore spécialiser un brin le mixin :

```suggestion
      this.router.transitionTo('login');
```

Vu que le nom de la route `terms-of-service` est lui écrit en dur juste en-dessous, autant être cohérent.

Reply:
Je ne suis pas sûr que l'on puisse supprimer l'attribut "authenticationRoute" car il peut être surchargé par les routes qui étendent le mixin (voir start-or-resume.js)

Reply:
Bien vu, je n'avais même pas été vérifier. Il y a effectivement un modification de `authenticationRoute` dans `start-or-resume.js`  :https://github.com/1024pix/pix/blob/3d3906f0de7f5dd1fa10c853f713ea1801075944/mon-pix/app/routes/campaigns/start-or-resume.js#L35-L38

Je me demande si ce `set` a vraiment un effet. En tout cas il paraît en doublon/conflit avec le `transitionTo` qui suit.

OK pour garder le `authenticationRoute` en attendant que ça soit tiré au clair…